convert_tendl_rand.py

```diff
@@ -0,0 +1,361 @@
+        "isItENDF": True,
+        "gunzip": True,
+    },
+    "Si29": {
+        "release": "ENDF2017",
+        "filename": "Si029",
+        "webname": "Si29",
+        "fileNum": 600,
+        "downSize": 100,
+        "fileSize": 425,
+        "isItENDF": True,
+        "gunzip": True,
+    },
+    "Si30": {
+        "release": "ENDF2017",
+        "filename": "Si030",
+        "webname": "Si30",
+        "fileNum": 600,
+        "downSize": 80,
+        "fileSize": 337,
+        "isItENDF": True,
+        "gunzip": True,
+    },
+    "Fe54": {
+        "release": "ENDF2017",
+        "filename": "Fe054",
+        "webname": "Fe054",
+        "fileNum": 501,
+        "downSize": 320,
+        "fileSize": 1300,
+        "isItENDF": True,
+        "gunzip": False,
+    },
+    "Fe56": {
+        "release": "ENDF2017",
+        "filename": "Fe056",
+        "webname": "Fe056",
+        "fileNum": 614,
+        "downSize": 410,
+        "fileSize": 1600,
+        "isItENDF": True,
+        "gunzip": False,
+    },
+    "Fe57": {
+        "release": "ENDF2017",
+        "filename": "Fe057",
+        "webname": "Fe057",
+        "fileNum": 733,
+        "downSize": 525,
+        "fileSize": 2000,
+        "isItENDF": True,
+        "gunzip": False,
+    },
+    "Fe58": {
+        "release": "ENDF2017",
+        "filename": "Fe058",
+        "webname": "Fe058",
+        "fileNum": 501,
+        "downSize": 280,
+        "fileSize": 1100,
+        "isItENDF": True,
+        "gunzip": False,
+    },
+    "Na23": {
+        "release": "ENDF2017",
+        "filename": "Na023",
+        "webname": "Na23",
+        "fileNum": 832,
+        "downSize": 130,
+        "fileSize": 544,
+        "isItENDF": True,
+        "gunzip": False,
+    },
+    "Pu240": {
+        "release": "ENDF2017",
+        "filename": "Pu240",
+        "webname": "Pu240",
+        "fileNum": 630,
+        "downSize": 200,
+        "fileSize": 900,
+        "isItENDF": True,
+        "gunzip": False,
+    },
+}
+
+if not formatOnly:
+
+    downloadFileSize = 0
+    uncompressedFileSize = 0
+    NumOfFiles = 0
+
+    for i in list_:
+        downloadFileSize += nuclide_details[i]["downSize"]
+        uncompressedFileSize += nuclide_details[i]["fileSize"]
+        NumOfFiles += nuclide_details[i]["fileNum"]
+
+    downloadSize = "{} MB".format(downloadFileSize)
+    uncomFileSize = "{} MB".format(uncompressedFileSize)
+    if downloadFileSize > 1000:
+        downloadSize = "{} GB".format(downloadFileSize / 1000)
+    if uncompressedFileSize > 1000:
+        uncomFileSize = "{} GB".format(uncompressedFileSize / 1000)
+
+
+    download_warning = """
+    WARNING: This script will download {} of 
+    data, which is {} of data when processed. 
+    This corresponds to {} random crossections.
+
+    The nuclides to be processed are: 
+    {}
+
+    Are you sure you want to continue? ([y]/n)
+    """.format(
+        downloadSize, uncomFileSize, NumOfFiles, list_
```

Comment:
PEP8: For local variables, you should generally be using `names_with_underscores` rather than camelCase